website/events/api/v2/views.py

```diff
@@ -124,6 +125,11 @@ def get_queryset(self):
             ).select_related("member__profile")[: self.event.max_participants]
         return EventRegistration.objects.none()
 
+    def get_serializer(self, registrations=None, *args, **kwargs):
```

Comment:
I overrode get_serializer in several places, I'm not sure it's the best function to do this but get_queryset is called twice for some reason.